### PR TITLE
Add ability to specify CPU

### PR DESCRIPTION
**Pull request by fridex**
Based on work done by @ace2107 
Replaces: #77

### CODE REVIEW

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
@goern there is a need to update deployment to also include newly added -with-cpu job and build templates.

File: amun/swagger.yaml [hunk 1]
```diff
@@ -205,23 +205,47 @@ definitions:
...
+                    type: integer
+                    description: CPU family number from CPU proc info.
+                    example: 6
+                  cpu_model:
+                    type: integer
+                    description: Model number of CPU from CPU proc info.
+                    example: 94
+                  physical_cpus:
+                    type: integer
+                    description: Number of physical physical CPUs.
```
Comment (goern) [inline amun/swagger.yaml]: cutnpasteerror?

File: amun/swagger.yaml [hunk 2]
```diff
@@ -236,12 +260,36 @@ definitions:
...
```
Comment (goern) [inline amun/swagger.yaml]: #2

File: openshift/inspectJobWithCPU-template.yaml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  - name: AMUN_MEMORY
+    description: Memory requested for build
+    displayName: Build Memory
+    default: 256Mi
+    required: true
+
+  - name: CPU_FAMILY
+    description: Family number of CPU.
+    displayName: CPU Family
+    required: true
```
Comment (goern) [inline openshift/inspectJobWithCPU-template.yaml]: can we provide sane defaults? @sub-mod

Comment (sub-mod) [reply openshift/inspectJobWithCPU-template.yaml]: "cpufamily": "6", "cpumodel": "94" , "processor" : "Intel-Xeon-Processor-Skylake-IBRS" , cpu_physicalcpus : 32

Comment (fridex) [reply openshift/inspectJobWithCPU-template.yaml]: Adjusted.

**Comment by fridex** [comment]
recheck

**Pull request merged**